### DISCUSSION TITLE
Are we able now to specify model temperature for transformers and AWQ models?

### DISCUSSION THREAD

**Discussion by aalyousfi** [outlines-dev/outlines#509] [Q&A] [answered]
Hello,

I asked about temperature a month ago and I understood that it was a work in progress. So just wanted to check if it's added now? And if not, can you please let me know when you expect it to be added? @rlouf 

Thank you!

**Comment by rlouf** [comment, marked as answer]
As maintainers we have limited time to triage issues, reply, review PRs, implement new features and prioritise requests. Adding temperature, top-p and top-k are in our backlog of features to implement but currently not the highest priority feature. Please bear with us while we work through the list. PRs are always welcome and we make it a priority to review them quickly.

**Reply by aalyousfi** [reply]
@rlouf No problem, just wanted to know if they have been made available.
Thank you very much for your valuable work!
